app/controllers/ViewTransfersController.scala

```diff
@@ -24,28 +52,74 @@ class ViewTransfersController @Inject() (val consignmentService: ConsignmentServ
         case None        => Nil
       }
     } yield {
-      Ok(views.html.viewTransfers(consignments, request.token.name, request.token.email))
+      val consignmentType = if (request.token.isJudgmentUser) "judgments" else "standard"
+      Ok(views.html.viewTransfers(consignments, request.token.name, request.token.email, consignmentType))
     }
   }
 
-  private def createView(edges: Option[Edges]): Option[ConsignmentTransfers] = {
+  private def createView(edges: Option[Edges]): Option[ConsignmentTransfers] =
+    edges.map { edge =>
+      val userActions = convertStandardStatusesToUserActions(edge.node.currentStatus, edge.node.consignmentid.get, edge.node.consignmentReference)
 
-    val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy")
-    edges.map(edges =>
       ConsignmentTransfers(
-        edges.node.consignmentid,
-        edges.node.consignmentReference,
-        getConsignmentStatus(edges.node.currentStatus),
-        edges.node.exportDatetime.map(_.format(formatter)).getOrElse(""),
-        edges.node.createdDatetime.map(_.format(formatter)).getOrElse(""),
-        edges.node.totalFiles
+        edge.node.consignmentid,
+        edge.node.consignmentReference,
+        userActions.transferStatus,
+        statusColours(userActions.transferStatus),
+        userActions,
+        edge.node.exportDatetime.map(_.format(formatter)).getOrElse("N/A"),
+        edge.node.createdDatetime.map(_.format(formatter)).getOrElse(""),
+        edge.node.totalFiles
       )
-    )
-  }
+    }
+
+  private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
 
-  private def getConsignmentStatus(currentStatus: CurrentStatus): String = {
-    if (currentStatus.`export`.contains("Completed")) "Exported" else "In Progress"
+  // scalastyle:off cyclomatic.complexity
+  private def convertStandardStatusesToUserActions(statuses: CurrentStatus, consignmentId: UUID, consignmentRef: String) = {
```

Comment:
I would actually not make this private, so you can write specific unit tests for this function. The logic is complex, and it justifies having unit tests for it, to help developers in the future understand the expected behaviours clearly.